tests.py

```diff
@@ -163,6 +157,55 @@ def test_roomExits(self):
                                 f"{randDir} should be a "
                                 f"valid exit in {room.name}")
 
+    def test_roomVerbs(self):
+        """
+        Make sure all rooms return a valid and non-empty response string
+        with function verbResponses.
+        """
+        # Verbs the game should recognize
+        validVerbs = [
+            "Look", "Look at", "Examine", "Go", "Exit", "Take", "Pick up",
+            "Help", "Inventory", "Savegame", "Loadgame", "Open", "Close",
+            "Hide", "Listen", "Peel", "Pull", "Use", "Drop", "Leave"
+            "Eat", "Shake", "Ring", "Flip"
+            ]
+        # Verbs the game doesn't recognize
+        invalidVerbs = [
+            "Sneeze", "Kick", "Catch"
+            ]
+        # Items that exist
+        validItems = [
+            "battery", "canOpener", "catFood", "flashlight", "key", "ladder",
+            "polaroid1", "polaroid2", "polaroid3", "silverBell", "tinCan"
+            ]
+        # Room features that exist
+        validFeatures = [
+            "window", "wall", "boxes", "cabinet"
+            ]
+        # Items/features that do not exist
+        invalidTargets = [
+            "cat", "dog", "paper"
+            ]
+
+        verbs = validVerbs + invalidVerbs
+        targets = validItems + validFeatures + invalidTargets
+
+        # Build tests
+        for verb in verbs:
+            for target in targets:
+                for room in self.testGame.rooms:
+                    verbTarget = room.verbResponses(verb, target)
+                    targetVerb = room.verbResponses(target, verb)
+                    verb = room.verbResponses(verb, "")
+                    target = room.verbResponses(target, "")
+                    results = [verbTarget, targetVerb, verb, target]
```

Comment:
It looks like you are reassigning "verb" and "target" within the room loop (lines 199 and 200). That means that after the first loop "verb" is changed from the actionVerb to a verbResponse. Same thing with target

Reply:
Yes, thank you, made some changes.